cmd/sql-main.go

```diff
@@ -84,16 +392,34 @@ func sqlSelect(targetURL, expression string, encKeyDB map[string][]prefixSSEPair
 	}
 
 	sseKey := getSSE(targetURL, encKeyDB[alias])
-	outputer, err := targetClnt.Select(expression, sseKey)
+	outputer, err := targetClnt.Select(expression, sseKey, selOpts)
 	if err != nil {
 		return err.Trace(targetURL, expression)
 	}
 	defer outputer.Close()
-
+	if len(csvHdrs) > 0 && writeHdr {
+		fmt.Println(strings.Join(csvHdrs, ","))
```

Comment:
Should this be removed ?

Reply:
no, it is writing the csv header to stdout

Reply:
Alright can you add a comment ?